src/components/TopBar/CallButton.vue

```diff
@@ -178,7 +178,15 @@ export default {
 		},
 
 		/**
-		 * Whether the to use text on button at mobile view
+		 * Whether to use text on button (e.g. at sidebar)
+		 */
+		hideText: {
+			type: Boolean,
+			default: false,
+		},
+
+		/**
+		 * Whether to use text on button at mobile view
```

Comment:
Let's also use it for chat sidebar in Files?

Reply:
It's already there, but for the TopBar. Button in the chat has enough space to show the text

Reply:
oops, pointed to the wrong line :
`<CallButton v-if="!isInCall"`

so call button can be removed 
![image](https://github.com/user-attachments/assets/a106d313-afab-42c5-b37d-21ff3c25d8ad)